HTML autocorrect attribute (#35692)
mdn/content: files/en-us/web/html/element/textarea/index.md

This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attributes).

- [`autocapitalize`](/en-US/docs/Web/HTML/Global_attributes/autocapitalize)

>   - : A string that indicates whether input text is automatically capitalized and, if so, in what manner.

Author: ```suggestion
  - : Controls whether inputted text is automatically capitalized and, if so, in what manner.
```

- [`autocomplete`](/en-US/docs/Web/HTML/Attributes/autocomplete)

>   - : A string that indicates whether the value of the control can be automatically completed by the browser. Possible values are:

Author: I spend to much time in Web API land, where the type matters. Reverting those bits.

```suggestion
  - : This attribute indicates whether the value of the control can be automatically completed by the browser. Possible values are:
```

    - `off`: The user must explicitly enter a value into this field for every use, or the document provides its own auto-completion method; the browser does not automatically complete the entry.
    - `on`: The browser can automatically complete the value based on values that the user has entered during previous uses.
    - [`<token-list>`](/en-US/docs/Web/HTML/Attributes/autocomplete#token_list_tokens): An ordered set of space-separated autofill detail tokens, optionally preceded by a sectioning token, a billing or shipping grouping token, and/or a token identifying the type of recipient.

    `<textarea>` elements that don't specify the `autocomplete` attribute inherit the `autocomplete` `on` or `off` status set on the `<textarea>`'s form owner. The form owner is either the {{HTMLElement("form")}} element that this `<textarea>` element is a descendant of or the form element whose `id` is specified by the `form` attribute of the input element. For more information, see the [`autocomplete`](/en-US/docs/Web/HTML/Element/form#autocomplete) attribute in {{HTMLElement("form")}}.

- [`autocorrect`](/en-US/docs/Web/HTML/Global_attributes/autocorrect)

>   - : A string that indicates whether to activate automatic spelling correction and processing of text while the user is editing this `textarea`.

Author: ```suggestion
  - : Controls whether to activate automatic spelling correction and processing of text while the user is editing this `textarea`.
```
